Adress_class.py

```diff
@@ -1,37 +1,28 @@
-
-
-class Adress:
+class Address:
     """
     A class for generating random addresses in Cologne.
 
     Attributes:
-        adress_list (list): A list to store generated addresses.
+        address_list (list): A list to store generated addresses.
 
     Methods:
-        __init__(self):
-            Initialize an Adress object.
-
-        generate_random_address(self):
-            Generate a random address and store it in the address list.
-
+        generate_random_address(cls):
+            Generate a random address without creating an object.
+            
+            Returns:
+                str: A randomly generated address in the format "Boissereestraße <number>, 50674 Köln".
     """
-    
-    def __init__(self):
-        """
-        Initialize an Adress object.
 
+    address_list = []  
+
+    @classmethod
+    def generate_random_address(cls):
         """
-        self.adress_list = []
-    
-    def generate_random_address(self):
-        """
-        Generate a random address and store it in the address list.
+        Generate a random address without creating an object.
 
         Returns:
             str: A randomly generated address in the format "Boissereestraße <number>, 50674 Köln".
-
         """
-        random_address = f"Boissereestraße {len(self.adress_list) + 1}, 50674 Köln"
-        self.adress_list.append(random_address)
+        random_address = f"Boissereestraße {len(cls.address_list) + 1}, 50674 Köln"
+        cls.address_list.append(random_address)
```

Comment:
Very good making the `address_list` a static property o the class and accessing the class rather than the object instance in this method now.
Also marking this with `@classmethod` is a very nice approach.